### PR TITLE
Update redirect-traffic.mdx

### PR DESCRIPTION
**Pull request by gitrgoliveira**

### CODE REVIEW

File: website/content/commands/connect/redirect-traffic.mdx [hunk 1]
```diff
@@ -36,6 +36,8 @@ Usage: `consul connect redirect-traffic [options]`
 
 #### Options for Traffic Redirection Rules
 
+- `-consul-dns-ip` - The IP address of the Consul DNS resolver. If provided, Consul DNS queries will be redirected to the provided IP address for resolving.
```
Comment (blake) [inline website/content/commands/connect/redirect-traffic.mdx:39]:
```suggestion
- `-consul-dns-ip` - The IP address of the Consul DNS resolver. If provided, DNS queries will be redirected to the provided IP address for name resolution.
```
An iptables rule will be installed that redirects all DNS queries to the provided IP address, not just queries for the `.consul` TLD.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/572803.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 7c31acca72de4ddd154e8788930024e6b4ec892e onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 7c31acca72de4ddd154e8788930024e6b4ec892e onto `release/1.11.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 7c31acca72de4ddd154e8788930024e6b4ec892e onto `release/1.10.x` succeeded!

**Pull request merged**